Python/clinic/sysmodule.c.h

```diff
@@ -1192,7 +1192,7 @@ PyDoc_STRVAR(sys_deactivate_stack_trampoline__doc__,
 "deactivate_stack_trampoline($module, /)\n"
 "--\n"
 "\n"
-"Dectivate the perf profiler trampoline.");
+"Dectivate stack profiler trampoline backend.");
```

Comment:
Does this deactivate the (only) active backend?  (and what happens if there are not active backends?)

There is some asymmetry in the docstring of the previous function, which activate the *backend* passed as an argument, and this function that deactivates *a* backend (the active one?).

Reply:
> Does this deactivate the (only) active backend?

It deactivates any backend. Right now is only 1

Reply:
> Maybe we should clarify this a bit better on the docstring by mentioning "the active backend"

+1